src/components/templates/PartyMap/components/Map/MapRoom.module.scss

```diff
@@ -6,14 +6,24 @@
 @use "scss/attendee/effects/atom";
 @use "scss/attendee/effects/rounded";
 @use "scss/attendee/layout";
+@use "scss/attendee/breakpoint";
+
+$-title-height: 42px;
 
 .MapRoom {
-  display: flex;
-  flex-direction: column;
-  overflow: hidden;
-  max-width: layout.$w-sidebar;
+  @media (max-width: calc(breakpoint.$md - 1px)) {
+    // Override inline styles
+    height: auto !important;
+    width: auto !important;
```

Comment:
Why the need to override? `@debt` comment would be nice if there is possible removal of `!important` in the future, but would be good to at least explain why, not what.